workflows/phylogenetics/wf_augur_prep.wdl

```diff
@@ -8,22 +8,22 @@ workflow augur_prep {
     File assembly
     String? collection_date
     String? country
-    String? state
-    String? continent
+    String? division
+    String? region
     String? pango_lineage
-    String? nextclade_clade
```

Comment:
I'm thinking now, and maybe this should be just clade? It technically can be any typing information, no?